dlp/api/DlpTest/InspectTests.cs

```diff
@@ -15,139 +15,136 @@
 using GoogleCloudSamples;
 using Xunit;
 
-namespace DlpTest
+public class InspectTests : IClassFixture<DlpTestFixture>
```

Comment:
Consider using it as a Collection fixture so that we truly have one per run. https://xunit.net/docs/shared-context

Reply:
Ack - will likely do this in a future PR if it is necessary. No heavyweight init here yet.